the22mastermind/questioner
# Ft admin 162701470 endpoint

#### What does this PR do?
Add create a meetup functionality with database
#### Description of Task to be completed?
Create a new meetup
#### How should this be manually tested?
1. After cloning repo, cd into it and run 'npm run start'
2. Using POSTMAN enter in URL: [POST] http://localhost:5000/api/v1/meetups/ and hit send, you should be prompted to enter the required fields.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#162701470 https://www.pivotaltracker.com/story/show/162701470
#### Screenshots (if appropriate)
#### Questions:
